src/note_encryption/compact_action.rs

```diff
@@ -70,8 +74,10 @@ impl<D: OrchardDomainCommon> ShieldedOutput<OrchardDomain<D>> for CompactAction<
         None
     }
 
+    // FIXME: shouldn't it return None like enc_ciphertext does?
```

Comment:
no, `enc_ciphertext` should not be used for Orchard. It was used only by sapling.

Reply:
Got it, removed the comment.